Makefile

```
@@ -901,10 +863,12 @@ deploy/bootstrap:
 # Deploy local images fast for development
 deploy/dev-fast: image/build deploy/dev-fast/fleet-manager deploy/dev-fast/fleetshard-sync
```

`make deploy/dev` now uses `image/build/local`
`make deploy/dev-fast` uses `image/build`

Is that intentional?

---

```suggestion
deploy/dev-fast: deploy/dev-fast/fleet-manager deploy/dev-fast/fleetshard-sync
```

There was still one instance left. 😅

---

:tada: fixed